From 36db194ae95e35cc7b8f431ab984780dea24656d Mon Sep 17 00:00:00 2001
From: Mathieu Trudel-Lapierre <mathieu.tl@gmail.com>
Date: Wed, 20 Jul 2011 23:56:48 -0500
Subject: [PATCH 2/2] libnm-util: default to allowing IPv6 connections to fail

Allow IPv6 connections to fail by changing the default value of the may-fail
property for ipv6 settings to TRUE.

This makes sure connections created manually using nm-connection-editor
will default to having the "Require ..." checkbox unchecked; in other words,
IPv6 timing out will still allow the interface to be configured if
IPv4 succeeds.
---
 libnm-util/nm-setting-ip6-config.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/libnm-util/nm-setting-ip6-config.c b/libnm-util/nm-setting-ip6-config.c
index e911639..4dc8f46 100644
--- a/libnm-util/nm-setting-ip6-config.c
+++ b/libnm-util/nm-setting-ip6-config.c
@@ -1056,7 +1056,7 @@ nm_setting_ip6_config_class_init (NMSettingIP6ConfigClass *setting_class)
 						   "this property to TRUE allows the overall network "
 						   "configuration to succeed if IPv6 configuration "
 						   "fails but IPv4 configuration completes successfully.",
-						   FALSE,
+						   TRUE,
 						   G_PARAM_READWRITE | G_PARAM_CONSTRUCT | NM_SETTING_PARAM_SERIALIZE));
 }
 
-- 
1.7.6

